SPECS/boost/boost.spec

```diff
@@ -48,6 +48,8 @@ The boost-static package contains boost static libraries.
 %defattr(-,root,root)
 %license LICENSE_1_0.txt
 %{_libdir}/libboost_*.so.*
+%{_libdir}/cmake/*.cmake
+%{_libdir}/cmake/*/*.cmake
```

Comment:
I don't think the `*.cmake` files make any sense to include for `boost`. I'd follow what we've been doing for now (and it seems Fedora and other distros are doing a similar thing), where we limit ourselves to the shared libraries and the licenses.

Here's a suggestion of what to add in the `%install` sections instead of these two lines here:

```bash
rm -rf %{buildroot}%{_libdir}/cmake
```

Reply:
removed